tests/integ/scala/test_datatype_suite.py

```diff
@@ -381,7 +381,7 @@ def test_structured_dtypes_select(structured_type_session, examples):
     query, expected_dtypes, expected_schema = examples
     df = _create_test_dataframe(structured_type_session)
     flattened_df = df.select(
-        df.map["k1"].alias("value1"),
+        df.Map["k1"].alias("value1"),
```

Comment:
this change implies any customers who are using this way to access column names `map` will break. This is a breaking change then.
cc: @sfc-gh-yixie

Reply:
@sfc-gh-aalam  perhaps we can change the name of this new method to avoid this situation. For example, we could use `apply_func`. Do you agree with this alternative?